app/templates/toezicht/inzendingen.hbs

```diff
@@ -36,6 +44,11 @@
 
               <label class="label--subtle form__label">Type document</label>
               {{besluit-type-select value=besluitTypeId onSelectionChange=(action (mut besluitTypeId)) class="ember-power-select--small"}}
+              
+              {{#if (eq besluitTypeId  '5b3955cc006323233e711c482f3a6bf39a8d3eba6bbdb2c672bdfcf2b2985b03')}}
```

Comment:
It's better to compare on the URI since that's the most stable property of a resource.
To make the template more human readable, it may be a good to define a computed property that explains the purpose of this comparison (e.g. `isRegulation`)